packages/wallet/ui/src/components/ConnectionSettingsDialog.jsx

```diff
@@ -136,12 +136,12 @@ const ConnectionSettingsDialog = ({
             }
           }}
         >
-          <MenuItem value="main">Main</MenuItem>
+          <MenuItem value="main">Mainnet</MenuItem>
           <MenuItem value="testnet">Testnet</MenuItem>
           <MenuItem value="devnet">Devnet</MenuItem>
-          <MenuItem value="localhost">localhost</MenuItem>
+          <MenuItem value="localhost">Localhost</MenuItem>
```

Comment:
I wanted it lower caps since it's not a network name and the actual host name

Reply:
It just looked a bit jarring since the others are caps. The others are titles/labels, not hostnames, so I prefer having this consistency.